src/components/leftSide/RemoveFriendBtn.tsx

```diff
@@ -0,0 +1,36 @@
+import { getSocket } from "socket/socket";
+import * as S from "./style";
+import { useQueryClient } from "@tanstack/react-query";
+import { ChatRoomResponse } from "socket/active/chatEventType";
+import { useEffect } from "react";
+
+export default function RemoveFrendBtn(props: { username: string }) {
+  const socket = getSocket();
+  const queryCli = useQueryClient();
+
+  const listener = (res: ChatRoomResponse) => {
+    if (res.status === "approved") {
+      alert("친구 취소 했습니다.");
+      queryCli.invalidateQueries(["profile", props.username]);
```

Comment:
```suggestion
      queryCli.refetchQueries(["profile", props.username]);
```

음.....정확한건 아니지만! 아마 invalidate해서 데이터가 상한(stale) 상태가 되었을 때, 다시 fetch하기 위한 조건(마운트, 포커싱 등)에 충족하지않아서 간헐적으로 refetch가 안되는게 아닐까 싶어요!
그래서 여기를 바로 refetch 시키면 해결되지않을까? 싶습니다